tests/unit/test_charm.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Tony Meyer
+# See LICENSE file for licensing details.
+#
+# Learn more about testing at: https://juju.is/docs/sdk/testing
+
+import ops
+import ops.testing
+import pytest
+from charm import TakahēOperatorCharm
+
+
+@pytest.fixture()
+def harness():
+    harness = ops.testing.Harness(TakahēOperatorCharm)
+    harness.begin()
+    yield harness
+    harness.cleanup()
+
+
+def test_pebble_ready_web(harness):
```

Comment:
These are the unit tests you get when running `charmcraft init --profile=kubernetes`, except (a) converted to pytest, and (b) duplicated for the two containers and with the status changed.

I probably could delete them, yes. I had originally thought that I'd write Harness and Scenario tests side-by-side to compare, but I did a bunch of that type of work already elsewhere and don't feel a lot of desire to do more of it! In terms of maintaining this going forward, and having it as an example of a well-organised charm, one form of unit tests would be cleaner. If there's something that can't be done with Scenario then it would be better to fix that than add a Harness test.